index.md

```diff
@@ -1335,6 +1335,55 @@ One thing for bot authors to note is that the `NOTICE` message may be interprete
 
 ## User-Based Queries
 
+### WHO message
+
+         Command: WHO
+      Parameters: [ <mask> [ "o" ] ]
+
+This command is used to query a list of users who match the provided mask.
+The server will answer this command with zero, one or more [`RPL_WHOREPLY`](#rplwhoreply-352), and end the list with [`RPL_ENDOFWHO`](#rplendofwho-315).
+
+The mask can be one of the following:
+
+* A channel name, in which case the channel members are listed.
+* An exact nickname, in which case a single user is returned.
+* A mask pattern, in which case all visible users whose host, server, real name or nickname matches are listed.
+* Absent or set to "0", in which case all visible users are listed.
+
+Visible users are users who aren't invisible ([user mode `+i`](https://modern.ircdocs.horse/#invisible-user-mode)) and who don't have a common channel with the requesting client.
+Servers MAY filter or limit visible users replies arbitrarily.
+
+If the "o" flag is given, only server operators are returned.
```

Comment:
Ergo actually ignore this, I opened an issue on their BT: https://github.com/ergochat/ergo/issues/1832

Reply:
This sounds like an implementation bug which needn't be mentioned here.

Reply:
Actually it's a design choice (see https://github.com/ergochat/ergo/issues/1730#issuecomment-968333055), but I agree it should not be mentioned here

Reply:
The ergo bug has been fixed in https://github.com/ergochat/ergo/commit/2038763e106696e5fefb32c936ac498d42d032b5 by returning an empty list.